Gameplay/entities/player/PlayerController.cpp

```diff
@@ -705,6 +705,18 @@ void Hachiko::Scripting::PlayerController::ChangeWeapon(unsigned weapon_idx)
 	}
 }
 
+void Hachiko::Scripting::PlayerController::ReloadAmmo(unsigned ammo)
+{
+	if (0 < ammo && ammo <= MAX_AMMO) 
```

Comment:
```suggestion
	if (ammo <= MAX_AMMO) 
```
No needs to check for smaller than zero as ammo is unsigned.